### ISSUE TITLE
How to download the original caffe keras model which MobileOpenPose.mlmodel converted from?

### ISSUE THREAD

**Issue by stayHF** [infocom-tpo/SwiftOpenPose#3]
Thanks for your great job first.
I have success run your swift demo on pose detect. 
Can you tell me how to download the original caffe keras model which MobileOpenPose.mlmodel converted from? 
  
Thanks.

**Comment by mbotsu** [comment]
Thank you for trying.

We do not provide models such as Keras and Caffe.
The goal of this project is to run on mobile.
At the moment, we are thinking about improving the use case, speed and accuracy, and there are no plans to increase the types of models we offer.

# You can try tf-openpose converting. 
## tf-openpose (instance_normalization supported)
https://www.dropbox.com/s/09xivpuboecge56/mobilenet_0.75_0.50_model-388003.zip?dl=0

## Convert From TensorFlow To Keras
https://github.com/infocom-tpo/tf-openpose/blob/b1de5e2cd630b4ebe30aae69735f3c7c70798224/convert/tensorToKeras.py

Thanks.
